docs/architecture/adr-050-sign-mode-textual.md

```diff
@@ -127,8 +127,32 @@ type SignDocTextual = []Screen
 We do not plan to use protobuf serialization to form the sequence of bytes
 that will be tranmitted and signed, in order to keep the decoder simple.
 We will use [CBOR](https://cbor.io) ([RFC 8949](https://www.rfc-editor.org/rfc/rfc8949.html)) instead.
+The encoding is defined by the following CDDL ([RFC 8610](https://www.rfc-editor.org/rfc/rfc8610)):
+
+```
+;;; CDDL (RFC 8610) Specification of SignDoc for SIGN_MODE_TEXTUAL.
```

Comment:
let's add an entry in this file's internal Changelog

Reply:
Done.